### PR TITLE
Lokalise: Translations update

### PR DESCRIPTION
**Pull request by EuropeanaDevelopment**
This pull request was initiated by Lokalise (user Lutz Biedinger) at 2020-01-16 15:22:09

### CODE REVIEW

File: lang/en.js [hunk 1]
```diff
@@ -82,15 +82,16 @@ export default {
...
-        "permission": "Permission",
+        "open": "Yes",
+        "restricted": "Yes, with conditions",
+        "permission": "Maybe, seek permission",
         "uncategorized": "Uncategorized"
       }
     },
     "THEME": {
-      "name": "Collection",
+      "name": "Category",
```
Comment (rwd) [inline lang/en.js]: Is this intentional?

File: lang/en.js [hunk 2]
```diff
@@ -82,15 +82,16 @@ export default {
...
       "options": {
+        "all": "All items",
```
Comment (rwd) [inline lang/en.js]: This is not used and should be removed.

File: lang/en.js [hunk 3]
```diff
@@ -394,7 +393,7 @@ export default {
   },
   "contentfulManual": {
```
Comment (rwd) [inline lang/en.js]: Is this ever used? I see no reference to it in the codebase.

File: lang/en.js (hunk 2 above)
Comment (lbiedinger) [reply lang/en.js]: I've removed the key from lokalize.

File: lang/en.js (hunk 1 above)
Comment (lbiedinger) [reply lang/en.js]: yes

File: lang/en.js (hunk 3 above)
Comment (rwd) [reply lang/en.js]: For reference, it's a place for translations in Lokalise which will then be input manually into Contentful, for now.

**Pull request merged**